src/pages/events/[eventId]/edit/index.page.tsx

```diff
@@ -13,11 +16,35 @@ export const getServerSideProps = getApplicationServerSideProps(
   async ({ req, query, queryClient, cookies }) => {
     const { eventId } = query;
 
-    await prefetchGetEventByIdQuery({
-      id: eventId,
-      req,
-      queryClient,
-    });
+    await Promise.all([
+      prefetchGetEventByIdQuery({
+        id: eventId,
+        req,
+        queryClient,
+      }),
+      prefetchGetEventPermissionsQuery({
+        req,
+        queryClient,
+        eventId: eventId,
+      }),
+    ]);
+
+    const permissions = queryClient.getQueryData([
+      'event-permissions',
+      { eventId },
+    ]);
+
+    if (
+      permissions?.permissions?.length === 0 ||
+      !permissions.permissions.includes(EventPermissionTypes.AANBOD_BEWERKEN)
```

Comment:
If you use desctructuring like suggested above, you can simplify this:
```suggestion
     permissions?.length === 0 ||
      !permissions.includes(EventPermissionTypes.AANBOD_BEWERKEN)
```